script/build.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const { execSync } = require('node:child_process')
+const settingData = require('./setting.json')
+
+// TODO update address before pop up online (update websiteMap in setting.json)
```

Comment:
what does it mean? do we need to update the address manually?

Reply:
It has been updated, just to remind myself